packages/peregrine/lib/talons/CheckoutPage/useCheckoutPage.js

```diff
@@ -89,16 +87,24 @@ export const useCheckoutPage = props => {
     }, [toggleDrawer]);
 
     const setShippingInformationDone = useCallback(
-        () => setCheckoutStep(CHECKOUT_STEP.SHIPPING_METHOD),
-        [setCheckoutStep]
+        () =>
+            checkoutStep === CHECKOUT_STEP.SHIPPING_ADDRESS &&
+            setCheckoutStep(CHECKOUT_STEP.SHIPPING_METHOD),
```

Comment:
If you have a function that doesn't return anything, such as this one, just open a function body and use an `if` statement.

Reply:
9deb452e0